src/sentry/apidocs/examples/tags_examples.py

```diff
@@ -25,6 +25,41 @@
     ],
 }
 
+SIMPLE_TAG_VALUES = [
+    {
+        "key": "chunkymonkey",
```

Comment:
Can we have some tags that are like real looking? 😅

Reply:
HAHA yeah I can update these - went too far with the ice cream theme huh